source/MaterialXCore/Value.h

```diff
@@ -73,7 +74,7 @@ class MX_CORE_API Value
     /// Create a new value instance from value and type strings.
     /// @return A shared pointer to a typed value, or an empty shared pointer
     ///    if the conversion to the given data type cannot be performed.
-    static ValuePtr createValueFromStrings(const string& value, const string& type);
+    static ValuePtr createValueFromStrings(const string& value, const string& type, ConstTypeDefPtr typeDefPtr = nullptr);
```

Comment:
For consistency, I'd recommend `typeDef` as the name of this argument, as we usually omit `ptr` from variable and argument names.